examples/pom.xml

```diff
@@ -48,6 +48,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-deploy-plugin</artifactId>
+                <version>3.1.1</version>
```

Comment:
I feel a bit funny about this one. Some context: we only include this bit so that we don't deploy the examples as part of our maven release process. That is - when these are being built inside of our own maven reactor. 

Optimally when a user copied these and used them, there'd be nothing at all in here about maven deployment. 

I suppose my question is - what warnings does the lack of version here produce? Do we see them 1/ when running within this project's overarching maven reactor, 2/ when running _separately_, that is, copied out of it, or 3/ in both cases?

Reply:
You see the following in both cases

<img width="1095" alt="Screenshot 2023-12-13 at 14 29 47" src="https://github.com/aws-powertools/powertools-lambda-java/assets/76766/870c4029-88a7-4c9d-97b9-90d6e8e8d070">

<img width="1051" alt="Screenshot 2023-12-13 at 14 31 19" src="https://github.com/aws-powertools/powertools-lambda-java/assets/76766/d2e7424f-5b85-4713-a2b2-a32b6f7b26d1">

Happy to leave it out - but thought it would reduce warnings (but there are lots of other warnings I have not touched that could be fixed such as shading warnings)- up to you!

Reply:
I agree with you - let's stick with the clean-up!